### DISCUSSION TITLE
GOSUMDB=off is not working

### DISCUSSION THREAD

**Discussion by XFNeo** [renovatebot/renovate#12235] [Request Help] [answered]
We have some own go modules which placed on your local artifactory. We pass GOSUMDB=off  and -e GOPROXY=https://artifactorycn.replaced.com/api/go/pd.saas-release.golang.group to renovatenot container.

When dependabot try to update them we got error:
<details><summary>Logs</summary>

```
 WARN: artifactErrors (repository=PROD.Platform.Cloud_Core/libs/go/rest-utils, branch=renovate/git.replaced.comprod.platform.cloud_core)
       "artifactErrors": [
         {
           "lockFile": "go.sum",
           "stderr": "Command failed: go get -d ./...\ngo: git.replaced.com/prod.platform.cloud_core/libs/go/security@v0.1.4: verifying go.mod: git.replaced.com/prod.platform.cloud_core/libs/go/security@v0.1.4/go.mod: reading https://artifactorycn.replaced.com/api/go/pd.saas-release.golang.group/sumdb/sum.golang.org/lookup/git.replaced.com/prod.platform.cloud_core/libs/go/security@v0.1.4: 404 Not Found\n"
         }
       ]
```

</details>
But when I run container locally

docker run --rm -it -e GOPROXY=https://artifactorycn.replaced.com/api/go/pd.saas-release.golang.group -e GOSUMDB=off artifactorycn.replaced.com:17064/renovate/renovate:28 bash

update our lib to new version in go.mod and do "go get -d ./..." I have not got such error.

**Comment by rarkins** [comment, marked as answer]
See #12243

**Reply by XFNeo** [reply]
Thanks! When version 28.7.1 will be available on docker hub?  Now 28.6.0 is the latest. https://hub.docker.com/r/renovate/renovate/tags

**Reply by viceice** [reply]
see:
- https://github.com/renovatebot/docker-renovate
- https://github.com/renovatebot/docker-renovate-full

**Reply by XFNeo** [reply]
It works. Thanks
